Tests/Functional/CompositeRepositoryTest.php

```diff
@@ -13,6 +13,8 @@
 
 class CompositeRepositoryTest extends PhpcrRepositoryTestCase
```

Comment:
Does CompositeRepository implement EditableInterface? That would be out of the scope of this PR I think. In anycase it should not extend a PHPCR test case.